rigs/heft-vscode-extension-rig/profiles/default/includes/eslint/mixins/friendly-locals.js

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+module.exports = {
```

Comment:
No need to include the non-flat variations + patch files for this rig, since it's new and already using the flat configs